client/src/pages/SummaryPage/SummaryPage.tsx

```diff
@@ -190,6 +236,11 @@ const DashboardPage: React.FC<IDashboardPageProps> = () => {
         <Grid item />
       </Grid>
 
+      <Grid container sx={{mt: '3rem', 'mb': '3rem', display: 'flex', justifyContent: 'space-around', 'mr': 'auto', 'ml': 'auto'}} width="90%">
+        <IndicatorChart data={avgWeightData} measureName="Average Weight per Block" />
+        <IndicatorChart data={avgTxsData} measureName="Average Transactions per Block" />
+        <IndicatorChart data={avgDifficultyData} measureName="Average Difficult per Block (in Trillions)" />
```

Comment:
spelling error

Reply:
@Kodylow  - Fixed the spelling issue - TY! Have a good night!